18-additional-resources.Rmd

```diff
@@ -96,23 +96,17 @@ Bryan, J. & Hestor, J. *Happy git and github for the useR*. Retrieved from [http
 
 *A short introduction to the caret package*. Retrieved from [https://cran.r-project.org/web/packages/caret/vignettes/caret.html]([https://cran.r-project.org/web/packages/caret/vignettes/caret.html)
 
-> The `caret` package is an abbreviated form of `Classification And REgression Trees`. It contains functions to streamline the model training process for complex regression and classification problems. The idea behind this package is to streamline the process for creating predictive models. The package contains tools for:
-
-    - data splitting
-    - pre-processing
-    - feature selection
-    - model tuning using resampling
-    - variable importance estimation
+> The `caret` package is an abbreviated form of `Classification And REgression Trees`. It consist of methods that solve regression and classification problems. It also consist of methods related to the initial data preprocessing activities like data cleaning, feature selection and model tuning for predictive analytics.
 
 *tidy data*. Retrieved from [https://tidyr.tidyverse.org/articles/tidy-data.html](https://tidyr.tidyverse.org/articles/tidy-data.html)
 
 Wickham et al. (2019). Welcome to the Tidyverse. *Journal of Open Source Software, 4*(43). 1686-1691. https://joss.theoj.org/papers/10.21105/joss.01686
 
-> The philosophy of ensuring data is clean such that each column contains data of just one type (or atomicity) and each row is an observation (made up of several atomic columns) is made possible in this paper. The principles of tidy data provide a standard way to organise data values within a dataset.
+> The philosophy of ensuring data is clean such that each column contains data of just one type (or atomicity) and each row is an observation (made up of several atomic columns) is made possible in this paper. An atomic data type simply means that the data type cannot be broken down further. For example, consider a person address to be like, "127, Napier Street, Kuala Lumpur 50603". Suppose we save this address to a variable called "house-address". If you look at this variable, it contains mixed-type data with both numeric and text values. So its important to break down the "house-address" variable such that it will contain data of just one type only. So it can be broken down into sub-variables like, `street-number`, `street-name`, `city-name`, `post-code`. This is known as data atomicity. Such atomicity leads to ensuring data is tidy and clean. For more details on this subject, please read this wonderful paper by Hadley Wickham (Wickham, H. (2014). Tidy data. Journal of Statistical Software, 59(10), 1-23.).
```

Comment:
minor, change "house-address" to `house-address`?

Reply:
thank you for the feedback. I have made the change.